builder.json

```diff
@@ -7,5 +7,18 @@
 
     "build_steps": [
         "npm install"
-    ]
+    ],
+    "variants" : {
+        "device_advisor": {
```

Comment:
Minor, but maybe rather than a variant, just make the test steps a top-level entry (if that works?) so that if we add additional non-da testing later, there wouldn't be confusion or new CI jobs needed, they could just be added to the end of the existing test steps?

Reply:
I just think with the variants, it is easier to disable/enable the tests. However, we can definitely just have the test at a top level.